### ISSUE TITLE
Make svelte:compiler a weak dependency

### ISSUE THREAD

**Issue by zodern** [rdb/svelte-meteor-data#14]
The `svelte:compiler` package requires apps to use a specific svelte version and warns when the app uses a different version. When apps or packages use `zodern:melte` instead, they can use the latest svelte version, but if they also use `rdb:svelte-meteor-data` they get shown the warning from `svelte:compiler` to use an older version of svelte:

```
WARNING: this package requires svelte@3.27.0 to be installed as npm peer dependency, but your package.json specifies:

  "svelte": "^3.44.2"

If you encounter error messages, consider running:

  meteor npm install --save svelte@3.27.0
```

If you make `svelte:compiler` a weak dependency, then apps can use the latest svelte version with `zodern:melte` and `rdb:svelte-meteor-data` without seeing the warning.

**Comment by skandragon** [comment]
Additionally, if one wants to use the latest compiler, it will break svelte-meteor-data.

**Comment by rdb** [comment]
The only issue I have with this is that this package really does depend on the npm `svelte` package being installed, and depending on `svelte:compiler` guarantees that.  Is there another way of guaranteeing that?

@skandragon how is this the case?  Right now there is a minimum version requirement.

**Comment by zodern** [comment]
> The only issue I have with this is that this package really does depend on the npm svelte package being installed, and depending on svelte:compiler guarantees that. Is there another way of guaranteeing that?

One option is using https://atmospherejs.com/tmeasday/check-npm-versions.

**Comment by rdb** [comment]
Just released 1.0.0 with this change.  Many thanks to @renanccastro for doing the work!

**Comment by renanccastro** [comment]
thanks for the help @rdb !
